concept/ConceptManager.java

```diff
@@ -81,8 +81,8 @@ public EntityType putEntityType(String label) {
     @Nullable
     @CheckReturnValue
     public EntityType getEntityType(String label) {
-        final Type type = getType(label);
-        if (type instanceof EntityType) return type.asEntityType();
+        final ThingType thingType = getThingType(label);
+        if (thingType instanceof EntityType) return thingType.asEntityType();
```

Comment:
```suggestion
        if (thingType.isEntityType()) return thingType.asEntityType();
```
There are other places in this file - let's change them all!

Reply:
Fixed in https://github.com/graknlabs/client-java/pull/211/commits/aa64cc93a7e0c3155a2b314f1dc9f400e88d41b8